lookout/style/format/analyzer.py

```diff
@@ -43,6 +45,54 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
         :param data: Contains "changes" - the list of changes in the pointed state.
         :return: List of comments.
         """
+        def get_error_description(vnode: VirtualNode, correct_y: int) -> str:
```

Comment:
Please rename `correct_y` to `predicted_y`or `y_pred` (to match your other function) to better make the distinction between the label (`node.y`) and the prediction.